packages/gatsby-theme-styleguide/src/gatsby-components/Menu/Menu.js

```diff
@@ -208,7 +222,19 @@ class Menu extends Component {
                 />
                 <NavContainer>
                   {this.buildMenu(this.state.menuArray, true)}
-
+                  <ListItem>
+                    <MenuItemContainer>
```

Comment:
@mohamedMok Currently, just the text and the icon are clickable. Can you make clickable the tile, please?